style.css

```diff
@@ -60,6 +60,61 @@ header{
     background-color: #87b0ed;
 }
 
+/* Mute button */
+#mute-btn {
+    float: right;
+    padding-left: 5px;
+    padding-right: 5px;
+}
+
+input[type=checkbox]{
+	height: 0;
+	width: 0;
+	visibility: hidden;
+    float: right;
+}
+
+label {
+	cursor: pointer;
+	text-indent: -45px;
+    text-shadow: #004d7a;
+	width: 40px;
+	height: 22px;
+	background: grey;
+	display: block;
+	border-radius: 100px;
+	position: relative;
+    float: right;
+
+}
+
+label:after {
+    content: '';
+    position: absolute;
+    top: 3px;
+    left: 3px;
+    width: 16px;
+    height: 16px;
+    background: #fff;
+    border-radius: 100px;
+    transition: 0.3s;
+    float: right;
+
+}
+
+input:checked + label {
+	background: #bada55;
+}
+
+input:checked + label:after {
+	left: calc(100% - 5px);
+	transform: translateX(-100%);
+}
+
+label:active:after {
+	width: 20px;
+}
```

Comment:
not a good idea to style the tag name directly, it can affect the other label and input